scripts/lint.sh

```diff
@@ -14,4 +14,9 @@ if ! yarn lint:report; then
   exit ${PUBLISH_TYPE_AND_RESULT_DIR_BUT_ALWAYS_FAIL}
 fi
 
+if ! yarn lint:dtslint; then
+  echo "DTS lint failed! Exiting..."
+  exit 1
+fi
+
```

Comment:
Shouldn't `lint:report` run this as well? Why call separately?

Reply:
Done